src/LetsEncrypt/ICertificateSource.cs

```diff
@@ -6,10 +6,18 @@
 using System.Threading;
 using System.Threading.Tasks;
 
-namespace McMaster.AspNetCore.LetsEncrypt.Internal
+namespace McMaster.AspNetCore.LetsEncrypt
 {
-    internal interface ICertificateSource
+    /// <summary>
+    /// Defines a source for certificates.
+    /// </summary>
+    public interface ICertificateSource
```

Comment:
FYI - this is okay to make this public for now, but I will probably make some changes before this gets released to nuget.org. I want to finish cleaning up the implementation that uses the X509Store so that it unifies into the use of one interface for providing certificates that the server can use.

Reply:
That's fine - I just needed to implement it.